x-pack/plugins/observability_solution/apm/server/routes/span_links/get_span_links_details.ts

```diff
@@ -120,14 +103,17 @@ async function fetchSpanLinksDetails({
     },
   });
 
+  const spanLinksDetails = response.hits.hits
+    .map((hit) => spanLinksDetailsMapping(hit.fields))
+    .filter((hits): hits is NonNullable<SpanLinksDetails> => !!hits);
```

Comment:
This needs to be in the mappings. There are a few places where ts complains about objects possibly being `undefined`